addons/web/static/src/core/notebook/notebook.scss

```diff
@@ -60,10 +60,6 @@
             &:first-child .nav-link {
                 border-top-width: 0;
             }
-
-            &:last-child .nav-link {
```

Comment:
Commit message mentions "re-introducing" something.

Can you point to a commit for reference?

It would also be worth mentioning that this vertical notebook is currently only used by the kanban examples dialog so it has a limited impact.